sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -255,10 +255,14 @@ opentitan_binary(
         "//hw/top_earlgrey:fpga_cw310",
         "//hw/top_earlgrey:sim_dv_base",
         "//hw/top_earlgrey:sim_verilator_base",
+        "//hw/top_earlgrey:silicon_creator",
     ],
     linker_script = ":ld_slot_a",
     manifest = ":manifest_standard",
-    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "test_key_0"},
+    rsa_key = select({
+        "//signing:test_keys": {"//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "test_key_0"},
+        "//conditions:default": {"//sw/device/silicon_creator/rom/keys/real/rsa:keyset": "earlgrey_a0_test_0"},
```

Comment:
Maybe I am missing something but I thought we were not using real keys by default?

Reply:
We are.  The `//signing:test_keys` condition is active by default.

This is an unfortunate side-effect of bazel config setting mappings: there is no "not equal to" match, so when the `//signing:token` flag is equal to it's default value of `//signing/tokens:local`, this activates the config setting `test_keys`.  If the user sets it to another value, such as `//signing/tokens:nitrokey`, it is not covered by any config settings and thus falls into `//conditions:default`.

I _think_ I can fix this to make the `select` statement make more sense, but I need to spend time developing that idea to see if it works the way I want.

Reply:
Ah I see, I didn't realize that. Thanks for the explanation.